test/parallel/test-process-getactivehandles.js

```diff
@@ -1,47 +1,11 @@
 'use strict';
 
-require('../common');
-const assert = require('assert');
-const net = require('net');
-const NUM = 8;
-const connections = [];
-const clients = [];
-let clients_counter = 0;
```

Comment:
I think we should leave the actual test in place since we're not removing these APIs yet.